homeassistant/components/huawei_lte/sensor.py

```diff
@@ -426,7 +426,7 @@ def _device_unique_id(self) -> str:
         return f"{self.key}.{self.item}"
 
     @property
-    def state(self) -> StateType:
+    def native_value(self) -> StateType:
         """Return sensor state."""
```

Comment:
Just an example from huawei_lte, this patch will leave lots of `native_value` methods behind whose docstring talks about returning the state, which feels a bit off. Better would be to have them mention returning the native value instead. And it also feels similarly off to have a return type named `StateType` for a method named like this.

Not _that_ big of a deal, but thought I'd mention it. Docstrings would likely need manual work to fix, and `StateType` thing probably not worth the churn.

Reply:
It's a valid point @scop, i can try to fix most/some docstrings in follow-up PRs depending on how many variations there really are.